kodi_addon_checker/check_addon.py

```diff
@@ -72,6 +74,9 @@ def start(addon_path, config=None):
         if len(addon_xml.findall("*//broken")) == 0:
             file_index = _create_file_index(addon_path)
 
+            if config.is_enabled("check_dependencies"):
```

Comment:
Please remove this setting. Should be not needed, now that the branch parameter is mandatory.